packages/wdio-cli/src/utils.js

```diff
@@ -196,3 +197,38 @@ export function addServiceDeps(names, packages, update) {
         }
     }
 }
+
+export function getCapabilities(arg) {
+    const capabilities = getPlatformCaps(arg.option)
+    let finalCapabilities = capabilities
+    if (capabilities.platformName === IOS_CONFIG.platformName) {
+        finalCapabilities = Object.assign( {},
+            capabilities,
+            {
+                deviceName: arg.device || IOS_CONFIG.deviceName,
+                platformVersion: arg.ver || null,
+                udid: arg.udid || null
+            }
+        )
+    }
+    return finalCapabilities
+}
+
+function getPlatformCaps(option) {
+    let platformConfig = {}
+    switch (option) {
+    case /.*\.(apk|app|ipa)$/.test(option) && option:
```

Comment:
Why check if option exists? Doing `.test` on a falsy value returns false. Also if option is a falsy value, the switch will default.